extensions/verticalpodautoscaler/tests/src/test/java/io/fabric8/verticalpodautoscaler/test/crud/V1VerticalPodAutoscalerCrudTest.java

```diff
@@ -72,7 +72,12 @@ void shouldLoadVerticalPodAutoscaler() {
         "    kind:       Deployment",
         "    name:       my-app",
         "  updatePolicy:",
-        "    updateMode: \"Auto\""
+        "    updateMode: \"Auto\"",
+        "  resourcePolicy:",
+        "    containerPolicies:",
+        "      - containerName: container",
+        "        controlledResources: [\"cpu\", \"memory\"]",
+        "        controlledValues: RequestsOnly"
```

Comment:
Could you please add an assert statement below to check whether resourceName is deserialized into VPA correctly?

Reply:
Umm, sorry. I meant assertion for `vpa.getSpec().getResourcePolicy().getContainerPolicies().get(0).getControlledResources()` (field for which this fix is)

Reply:
Sorry for the delay, done!